docker-proxy-rule-core/src/main/resources/docker-compose.proxy.yml

```diff
@@ -5,7 +5,7 @@ services:
     image: vimagick/dante:latest
```

Comment:
We should change this from latest so this doesn't happen in future.

Reply:
Oh, they only have latest :( :( https://hub.docker.com/r/vimagick/dante/tags

Reply:
Cool. Should I set this to the commit hash?

Reply:
Yeah, if that works I would do that

Reply:
ah, okay. Yeah I wasn't aware how to set it when no real versions exist 😅

Reply:
🌶️ 🌶️ 🌶️

Reply:
(To be clear, I wasn't aware if that works - I was just hoping it might)

Reply:
Trying the commit hash locally didn't seem to work, so we'll just go ahead with this in the short term.